marklogic-data-hub/src/test/java/com/marklogic/hub/dataservices/entitySearch/GetRecordTest.java

```diff
@@ -84,12 +88,13 @@ void testGetRecord() {
     }
 
     @Test
-    public void testRecordHistoryWithTwoProvenanceRecords() {
+    public void testRecordHistoryWithTwoProvenanceRecords() throws URISyntaxException {
         installProjectInFolder("test-projects/provenance-test" );
         String path = "test-projects/provenance-test/data/customers";
 
         FlowInputs inputs = new FlowInputs("inline");
-        inputs.setInputFilePath(getClass().getClassLoader().getResource(path).getPath());
+        File file = Paths.get(getClass().getClassLoader().getResource(path).toURI()).toFile();
```

Comment:
AbstractHubCentralTest has getFileFromClasspath in it, which uses ClassPathResource - a Spring class that I've never had issues with. 

TestObject has readInputStreamFromClasspath and readStringFromClasspath, both of which use ClassPathResource.

I think TestObject should have readFileFromClasspath, and ideally, it can use ClasspathResource as well. And then all of these changes that are using toURI can just use readFileFromClasspath.

Reply:
Also, getFileFromClasspath is only used in one place, so might as well get rid of it and just use readFileFromClasspath.